tasks/build-examples.js

```diff
@@ -16,6 +17,40 @@ var destDir = path.join(__dirname, '..', 'examples');
 var templatesDir = path.join(__dirname, '..', 'config', 'examples');
 
 /**
+ * Returns an array of classes that are explicitly required inside the source
+ * by calling 'goog.require(…)'
+ *
+ * @param {String} src The JavaScript sourcecode to search for goog.require.
+ * @returns {Array} An array of ol-classes that the source goog.requires.
+ */
+function getGoogRequires(src){
```

Comment:
The next linter will complain about the lack of space before the `{` here.

Really, it's not me being picky, it's the linter.